planter/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+  - BenTheElder
```

Comment:
nit: I think the indentation is wrong?

Reply:
This is copied from green house, if so I need to fix it there too 🤷‍♂️

Reply:
this appears to be a valid 2 space indent /shrug